feat(changelog): load-balancers-added-load-balancers-now-support 2023-01-06 (#1174)
scaleway/docs-content: changelog/january2023/2023-01-06-load-balancers-added-load-balancers-now-support.mdx

---
title: Load Balancers now support HTTP/2 and HTTP/3!
status: added
author:
    fullname: 'Join the #load-balancer channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-1-6
category: network
product: load-balancers
---

HTTP/2 offers significant improvements on HTTP/1.1 in terms of both performance and security. 

> HTTP/2 is now supported both at the frontend and the backend of Scaleway Load Balancers when TLS is enabled. Even better, you don't need to configure it to start using it. HTTP/2 is now enabled by default in all HTTP frontends and backends with TLS configured, and will automatically be used when supported by the client or backend server.

Reviewer: ```suggestion
HTTP/2 is now supported both at the frontend and the backend of Scaleway Load Balancers when TLS is enabled. No configuration is required to start using it. HTTP/2 is now enabled by default in all HTTP frontends, and backends with TLS configured and will automatically be used when supported by the client or backend server.
```

HTTP/3 is the next generation of HTTP protocol, built on top of QUIC. 

> HTTP/3 represents a big step forward for Internet performance and security. As this protocol is freshly baked and straight out of the oven, you have to enable HTTP/3 support if you wish to use it with a Scaleway Load Balancer. HTTP/3 is currently only supported at the frontend.

Reviewer 2: ```suggestion
HTTP/3 represents a big step forward for Internet performance and security. As this is a very new protocol, you have to enable HTTP/3 support if you wish to use it with a Scaleway Load Balancer. HTTP/3 is currently only supported at the frontend.
```

> HTTP/3 represents a big step forward for Internet performance and security. As this protocol is freshly baked and straight out of the oven, you have to enable HTTP/3 support if you wish to use it with a Scaleway Load Balancer. HTTP/3 is currently only supported at the frontend.

Reviewer: ```suggestion
HTTP/3 represents a big step forward for Internet performance and security. As this is a very new protocol, you have to enable HTTP/3 support if you wish to use it with a Scaleway Load Balancer. HTTP/3 is currently only supported at the frontend.
```
